index.html

```diff
@@ -48,22 +56,24 @@
                 <ul>
                     <li><a href="#">Квадрокоптеры</a></li>
                 </ul>                
-            </div>
+            </div> -->
         </header>
-        <main>
+        <main class="main-container">
             <h1 class="visually-hidden">Интернет магазин device</h1>            
-            <section class="slider">
+            <section class="section-slider">
                 <h2 class="visually-hidden">Реклама промо товаров</h2>
-                <div class="slider">
+                <div class="slider-container">
                     <div class="slider-item">
                         <div class="slider-img">
                             <img src=".//img/content-img/slider-1.png" alt="Три телескопические палки для селфи с мягкой рукояткой. Основной цвет - черный, декоративный цвет - синий">
```

Comment:
what for div wrapper for img? do you really need it?

Reply:
Removed div, corrected css